### DISCUSSION TITLE
Create a Community Spotlight on: Allan Regush 🙂

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#418] [Ideas] [answered]
Eddie created a blog post on his [dev.to account](https://dev.to/eddiejaoude) to spotlight a community member 2 months ago. I think @AllanRegush is a great candidate for this next edition on the series 😃.
IMO, he has been maintaining some repos on the community's organized very well, and dedicated time to help others.

What does everyone think about this?

**Comment by eddiejaoude** [comment, marked as answer]
Yes lets do it 👍 - I have written the next one that will be coming out this weekend (should have been last month, sorry I am behind 🤦‍♂️ )

I can add @AllanRegush to the list 👍 - thank you for the suggestion @BOLT04 ❤️
